force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,7 +111,17 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        String result = '';
+
+        if (a == null) {
+            return null;
+        } else if (Math.mod(a, 2) == 0) {
+            result = 'Even';
+        } else {
+            result = 'Odd';
+        }
+
+        return result; // Replace null with the variable you used to store the result
     }
```

Comment:
solid job here using Math.mod(). For problems like this in the real word I like to use a ternary operator. See if can solve this with  one if you want the practice.